e2e/__tests__/simple.spec.ts

```diff
@@ -0,0 +1,12 @@
+import configureTestCase from '../__helpers__/test-case';
+
+describe('Simple e2e test', () => {
+  it('should pass with babel 6', () => {
+    const testCase = configureTestCase('simple');
+    expect(testCase.run(0).status).toBe(0);
```

Comment:
The `0` (first parameter of `run()`) is optional and allow to have the output of the test run in the console in case the status code is not `0`. If not given, it'll simply fail without output, as before.